Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -606,65 +611,65 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
     /// up in the context, not from the arrays that was passed as arguments.
     ///
     /// - Parameters:
-    ///   - technologies: The list of temporary 'technology' pages.
+    ///   - TutorialTableOfContents: The list of temporary 'tutorial table-of-contents' pages.
```

Comment:
Casing is incorrect
```suggestion
    ///   - tutorialTableOfContents: The list of temporary 'tutorial table-of-contents' pages.
```

Reply:
Thanks. Fixed in https://github.com/swiftlang/swift-docc/pull/1039/commits/8287b2c7f965aec6c158f359f2d6005c5a0b7885